src/import/bundler.js

```diff
@@ -10,18 +10,14 @@
  * governing permissions and limitations under the License.
  */
 import esbuild from 'esbuild';
-
-function base64Encode(str) {
-  return Buffer.from(str).toString('base64');
-}
+import fs from 'fs';
 
 /**
- * Prepares the given import script by bundling it up and Base64 encoding it.
+ * Prepares the given import script by bundling it up and saving to the filesystem.
```

Comment:
Does this function actually save the file or just return the text content?

Reply:
At one point I had it saving the file but then moved to using Blob instead. The comment needs updating.